README.md

```diff
@@ -321,6 +321,9 @@ in your Home Manager configuration.
 Nix Flakes
 ----------
 
+As a NixOS module
+-----------------
```

Comment:
this probably needs to be a lower-level header than Nix Flakes (same with "For unprivileged setup")